src/plugins/mission/mission_impl.cpp

```diff
@@ -371,14 +371,14 @@ MissionImpl::convert_to_int_items(const std::vector<MissionItem>& mission_items)
 
                 MAVLinkMissionTransfer::ItemInt next_item{
                     static_cast<uint16_t>(int_items.size()),
-                    static_cast<uint8_t>(last_frame),
-                    MAV_CMD_NAV_LOITER_TIME,
+                    MAV_FRAME_MISSION,
+                    MAV_CMD_NAV_DELAY,
                     current,
                     autocontinue,
                     item.loiter_time_s, // loiter time in seconds
-                    NAN, // empty
-                    0.0f, // radius around waypoint in meters ?
-                    NAN, // don't change yaw
+                    -1, // no specified hour
+                    -1, // no specified minute
+                    -1, // no specified second
                     last_x,
                     last_y,
                     last_z,
```

Comment:
I don't understand those changes. Where do they come from? Did the MAVLink message change?

Reply:
@JonasVautherin see https://github.com/mavlink/MAVSDK/issues/1101, the idea was to switch over to MAV_CMD_NAV_DELAY.